Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -343,7 +343,7 @@ void addMeasurementToGx2fSums(Eigen::MatrixXd& aMatrixExtended,
     extendedJacobian.block<eBoundSize, 2>(0, deltaPosition) = jacPhiTheta;
   }
 
-  const BoundVector predicted = trackState.predicted();
+  const BoundVector smoothed = trackState.smoothed();
```

Comment:
`smoothed` might be a bit confusing at this point, since the GX2F relies on a prediction. The prediction at this point is also not really smoothed but just the raw (calibrated) prediction from the propagation. Maybe just keep `prediction`?